doc/nrf/ug_radio_fem.rst

```diff
@@ -201,26 +349,41 @@ This applies to all properties with a ``-gpios`` suffix, for both nRF21540 and S
 
 .. _ug_radio_fem_boards:
 
-Supported boards
+Hardware support
 ****************
 
 Two nRF21540 boards are available, showcasing the possibilities of the nRF21540 FEM:
 
 * :ref:`nRF21540 DK <nrf21540dk_nrf52840>`
-* nRF21540 EK, described in sections below
+* :ref:`ug_radio_fem_nrf21540_ek`
+
+For example SKY66112-11EK also has a 2-pin PA/LNA interface.
```

Comment:
```suggestion
For example, SKY66112-11EK also has a 2-pin PA/LNA interface.
```
TBH, I don't get this sentence here, it looks out of place. @peknis

Reply:
Maybe I would add this sentence before: Also various Skyworks front-end modules are supported

Reply:
Please check the new version